Packs/Azure-Enrichment-Remediation/Playbooks/Azure_-_Enrichment_README.md

```diff
@@ -0,0 +1,52 @@
+Given the IP address this playbook enriches Azure Compute, IAM, and User information.
+
+## Dependencies
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+This playbook does not use any sub-playbooks.
+
+### Integrations
+
+* Azure Compute v2
+* Azure Active Directory Identity And Access
+* Azure Active Directory Users
+
+### Scripts
+
+This playbook does not use any scripts.
+
+### Commands
+
+* azure-vm-get-public-ip-details
+* azure-vm-get-nic-details
+* azure-vm-get-instance-details
+* msgraph-identity-directory-roles-list
+* msgraph-identity-directory-role-members-list
+* msgraph-user-get
+
+## Playbook Inputs
+
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| AzureIP | Azure IP in alert | alert.remoteip | Required |
```

Comment:
Why the name of the input is Azure IP and not just an IP?

Reply:
Just to be specific, but it can be changed to just IP. 

For other enrichment / remediation packs we specified the cloud provider because the IP will specifically be tied to that provider.